assets/javascripts/discourse/templates/components/gamification-leaderboard-row.hbs

```diff
@@ -1,8 +1,8 @@
-<div class="user" id="leaderboard-user-{{rank.id}}">
+<div class="user {{if rank.currentUser "-highlight"}}" id="leaderboard-user-{{rank.id}}">
```

Comment:
Can we use `user-highlight` as the class here? A class starting with a `-` doesn't look right.

Reply:
it's my own usual -modifier class style I use but I changed it